packages/console-app2/src/components/ResourceList.tsx

```diff
@@ -26,7 +25,7 @@ function ResourceList ({ config }) {
       setApps(resources.filter(({ messageFqn }) => messageFqn === config.services.dxns.schema.fqn.app));
```

Comment:
Initially, just setRecords (single list of all records).

Reply:
This part is not changed yet because this is only a cleanup patch.
Would it be OK to continue with development in separate PR?